tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py

```diff
@@ -10,54 +12,128 @@ def __init__(self, *args, **kwargs):
-                                              'email': 'abdelmab@greenitglobe.com'}}])
-        self.cloudspaces.extend([{self.cs2: {'account': self.acc1,
-                                             'users': OrderedDict([('name', self.vdcuser),
-                                                                   ('accesstype', 'CXDRAU')])}}])
-        self.temp_actions = {'account': {'actions': ['install']},
-                             'vdcuser': {'actions': ['install']}, 'vdc': ['install']}
+        self.log('Create an cloudspace with non-exist account , should fail')
+        self.cloudspaces[self.cs1] =  {'account': self.random_string}
+        res = self.create_cs(openvcloud=self.openvcloud, accounts=self.accounts, vdcusers=self.vdcusers,
+                             cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
+        self.assertEqual(res,"account doesn't exist" )
 
-        self.log('Create 1 account and 2 cloudspaces, should succeed')
-        self.create_cs(openvcloud=self.openvcloud, vdcusers=self.vdcusers, accounts=self.accounts,
+        self.log('Create an cloudspace without providing account parameter, should fail')
+        self.cloudspaces[self.cs1] = {}
+        res = self.create_cs(openvcloud=self.openvcloud, vdcusers=self.vdcusers, accounts=self.accounts,
                        cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
+        self.assertEqual(res, 'account is required')
+
+        self.log('%s ENDED' % self._testID)
 
-        # wait till blueprint is executed
-        self.wait_for_service_action_status(self.cs2)
+    @parameterized.expand("one", "two")   
+    def test002_create_cloudspace(self, number ):
+        """ ZRT-OVC-002
+        *Test case for ...*
 
-        self.log('check that the cloudspaces have been created')
-        self.assertTrue(self.get_cloudspace(self.cs1))
-        self.assertTrue(self.get_cloudspace(self.cs2))
+        **Test Scenario:**
 
-        #check on the cloudspaces params
+        #. Create cloudspace with right parametrs , should succeed.
+        #. Create more 2 cloudspaces  with right parametrs , should succeed.
+        #. Check that the cloudspaces have been created.
+        
+        """
+        self.log('%s STARTED' % self._testID)
+        self.cloudspaces[self.cs1] = {'account': self.acc1}
+        if number == "two":
+            self.skipTest("https://github.com/openvcloud/0-templates/issues/49")
+            cs2 = self.random_string()
+            self.cloudspaces[cs2] = {'account': self.acc1}
+
+        self.log('Create %s cloudspaces, should succeed.'%number )
+        res = self.create_cs(openvcloud=self.openvcloud, vdcusers=self.vdcusers, accounts=self.accounts,
+                            cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
+        self.wait_for_service_action_status(self.cs1, res[self.cs1])
+
+        self.log('Check that the cloudspaces have been created.')
```

Comment:
https://github.com/openvcloud/0-templates/blob/master/templates/vdc/schema.capnp

- u need to pass and check on the params for the cloudspace that's defined in this capnp file .. u can do it here here or on the update task .. whatever